src/cheevos_libretro.c

```diff
@@ -37,8 +46,34 @@ static const rc_disallowed_setting_t _rc_disallowed_fbneo_settings[] = {
   { NULL, NULL }
 };
 
+static const rc_disallowed_setting_t _rc_disallowed_fceumm_settings[] = {
+  { "fceumm_region", ",PAL,Dendy" },
+  { NULL, NULL }
+};
+
 static const rc_disallowed_setting_t _rc_disallowed_gpgx_settings[] = {
```

Comment:
Seems this block will need to be duplicated for gpgx wide.
`genesis_plus_gx_wide_lock_on`
`genesis_plus_gx_wide_region_detect`

Reply:
Done.

I'm wondering if we should also block `genesis_plus_gx_wide_h40_extra_columns` and `genesis_plus_gx_wide_gg_extra`. In most cases, the extra background information won't be an advantage, but I could see a few places where it might be.

On a side note, I was able to get the extended Game Gear screen to work in RALibretro, but every time I tried to extend a Genesis game, the screen would stop rendering. I tried the smallest values, the recommended 10, and some specific values mentioned in the announcement blog post.

Reply:
I'd rather not move into blocking widescreen/overscan options. It may be an advantage in some games, but I don't believe it's significant enough in any game I've ever seen to deserve to be blocked in hardcore.

I can confirm the freeze issue in RALibretro, though the same setting works fine in RetroArch. Not sure why offhand.